tasks/install-matlab/v0/task.json

```diff
@@ -12,6 +12,16 @@
         "Minor": 1,
         "Patch": 0
     },
+    "inputs": [
+        {
+            "name": "release",
+            "type": "string",
+            "label": "Release",
+            "required": true,
+            "defaultValue": "latest",
+            "helpMarkDown": "MATLAB release to install. The value specified should follow the MATLAB release naming convention (e.g. R2020a). The latest release will be installed by default. Releases R2020a and above are available shortly after their general release on MathWorks.com."
```

Comment:
MATLAB release to install, specified as the full name MATLAB release version (for example, R2020a). By default, the installer installs the latest release.

(Tried to use active voice.)

What is the purpose of this sentence? I am not sure how it helps the user.

Releases R2020a and above are available shortly after their general release on MathWorks.com."

Reply:
I was hoping to convey that we are only offering R2020a and above, so users don't try to put in R2016a or something like that.

Reply:
OK. How about this?

MATLAB release to install, specified as the full name MATLAB release version (for example, R2020a). You can specify R2020a or a later release. By default, the installer installs the latest release of MATLAB.

Reply:
I like it but maybe we don't need the "specified..." part and we can avoid "installer"?

MATLAB release to install (for example, R2020a). You can specify R2020a or a later release. By default, the task installs the latest release of MATLAB.

Reply:
And maybe we don't even need the "for example..." since we have an example in the next sentence:

MATLAB release to install. You can specify R2020a or a later release. By default, the task installs the latest release of MATLAB.

Reply:
Reads great!